enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/iverksett/BehandlingStatistikkDto.kt

```diff
@@ -0,0 +1,24 @@
+package no.nav.familie.kontrakter.ef.iverksett
+
+import java.time.ZonedDateTime
+import java.util.UUID
+
+data class BehandlingStatistikkDto(
+        val behandlingId: UUID,
+        val personIdent: String,
+        val saksbehandlerId: String,
```

Comment:
Dette er "gjeldende" saksbehandler? Altså saksbehandler når det gjøres vilkårsvurdering, men beslutter når totrinnsvurdering godkjennes?

Reply:
Ja, det du sier er riktig.  Men jeg skjønner at det kan være forvirrende.... 
@charliemidtlyng @throndi  kunne vi kalle det for gjelendeSaksbehandlerId da?  :)

Reply:
Ja, i så fall så høres jo "gjeldendeSaksbehandlerId" mere riktig ut ? 🤔